src/meta-data/api/models/extract.py

```diff
@@ -21,16 +21,25 @@ def get_greenness(self, lat, lon, start_date, end_date):
         """
         Gets & returns the average greenness value at the specified coordinates
         for the specified time  period
+        Here is a more detailed interpretation of NDVI values:
+
+        -1.0 to 0.0: Water, clouds, snow, and ice
+        0.0 to 0.1: Bare rock, sand, and urban areas
+        0.1 to 0.2: Sparse vegetation, such as shrubs and grasslands
+        0.2 to 0.5: Moderate vegetation, such as crops and forests
+        0.5 to 0.7: Dense vegetation, such as tropical rainforests
+        0.7 to 1.0: Very dense vegetation, such as algae blooms
         """
         dataset = (
-            ee.ImageCollection("MODIS/006/MOD13Q1")
+            ee.ImageCollection("MODIS/061/MOD13Q1")
             .filter(ee.Filter.date(start_date, end_date))
             .mean()
         )
         location_geometry = ee.Geometry.Point(lon, lat)
         greenness_dict = dataset.reduceRegion(ee.Reducer.mean(), location_geometry, 90)
         mean_greenness = greenness_dict.get("NDVI")
         greenness = mean_greenness.getInfo()

```

Comment:
thanks @wabinyai  ,
Is it possible to reduce line 41 and 42 into one line?
From:
```
greenness = mean_greenness.getInfo()
greenness = greenness*0.0001
```

To:
`greenness = mean_greenness.getInfo()*0.0001`

Happy to learn in case I  could be missing something.
Thanks for this, great work and inspiring!

Reply:
Thanks for the suggestion, I have done that